oauth2client/django_util/apps.py

```diff
@@ -0,0 +1,7 @@
+from django.apps import AppConfig
+
+
+# pragma: NO COVER
```

Comment:
If I use the real Django test runner I think this would be covered, as it is I'm just using the usual nose tests and mocking more stuff. I could add django-nose and make a 'test app' that would give things like this coverage and also be better test coverage. When I was trying to fit this more into existing test patterns for the project I leaned against this, but since we might be splitting this into separate packages anyway it might make more sense to.